src/main/resources/application.yml

```diff
@@ -55,4 +55,4 @@ providerKeys:
 
 logging:
   level:
-    ROOT: ERROR
+    root: ERROR
```

Comment:
Provider source and provider keys related fields are still not following the naming convention.